setup.py

```diff
@@ -35,10 +35,15 @@
         "License :: OSI Approved :: MIT License",
         "Operating System :: OS Independent",
         "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
         "Topic :: Scientific/Engineering :: Visualization",
     ],
     packages=find_packages(),
     package_data={},
+    python_requires='~=3.7',
```

Comment:
I am not sure if this is necessary in this case, because matplotlib 3.5 dropped support for python 3.6, but using `getattr(matplotlib, "_all_deprecated", {})` should require python 3.7 (https://www.python.org/dev/peps/pep-0562). I haven't checked if this is correct but in any case, I think that it doesn't hurt to be explicit on the minimum python version supported!

Reply:
Actually `getattr` was there since Python 2.x (https://docs.python.org/2.7/library/functions.html#getattr) but there is little point that matplotlib-scalebar supports Python versions that matplotlib doesn't! Thanks.